pyproject.toml

```diff
@@ -101,3 +101,12 @@ force-exclude = '''
   )/
 )
 '''
+
+# Many deprecation warnings come from 3rd-party libraries and make the
+# output of pytest noisy. Since no action is going to be taken, hide those
+# warnings.
+[tool.pytest.ini_options]
+filterwarnings = [
+    "ignore:Deprecated call to.*",
+    "ignore:pkg_resources is deprecated as an API"
```

Comment:
Are these package specific or generic? Are we risking silencing warnings from inside MetricFlow?

Reply:
These apply to everything. We don't make calls to `warning.warn()`, so it's not silencing anything from MF right now. But it's a risk factor in the future. Not sure how to get around that though.